.github/workflows/wasm-sdk-build.yml

```
@@ -0,0 +1,105 @@
+  CARGO_PROFILE_RELEASE_LTO: false
+
+jobs:
+  build-wasm-sdk:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        
+      - name: Setup Rust toolchain
+        uses: actions-rs/toolchain@v1
```

actionlint (1.7.7)

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)